sms-one-time-code-format/README.md

```diff
@@ -15,10 +15,13 @@
 
 
 - [Introduction](#introduction)
-- [Deficiencies of the status quo](#deficiencies-of-the-status-quo)
-- [Goal](#goal)
-- [Non-goal](#non-goal)
+  - [Deficiencies of the status quo](#deficiencies-of-the-status-quo)
+  - [Goals](#goals)
+  - [Non-goals](#non-goals)
 - [Proposal](#proposal)
+  - [](#)
```

Comment:
What’s this line about?

Reply:
Noise. Fixed by running `doctoc` again in 4a932ca.